e2e/tests/query.feature

```diff
@@ -0,0 +1,391 @@
+Feature: Pocket Query Commands
+
+  Scenario: To show existing commands within the pocket query section
+	Given the user has a pocket client
+	When the user runs the command "query"
+	Then the user should be able to see standard output containing "Usage:"
+	And the pocket client should have exited without error
+
+  Scenario: To query an existing account
+	Given the user is running the network "single_node_network"
+	And the user has a pocket client
+	When the user runs the command "query account {{address}}" against validator 0
+	Then the user should be able to see standard output containing "/v1/query/account"
+	And the user should be able to see standard output containing "address"
+	And the pocket client should have exited without error
+
+  Scenario: To query an existing account, wrong address
+	Given the user is running the network "single_node_network"
+	And the user has a pocket client
+	When the user runs the command "query account {{address}} 0" against validator -1
+	Then the user should be able to see standard output containing "encoding/hex: invalid byte:"
+	And the pocket client should have exited without error
+
+  Scenario: Pocket Query Account address is invalid But Has valid Bytes
+	Given the user is running the network "single_node_network"
+	And the user has a pocket client
+	When the user runs the command "query account 2BBCA5DC9792C72AC3A2616910C4AAAA 0"
+	Then the user should be able to see standard output containing "Incorrect address length"
+	And the pocket client should have exited without error
+
+  Scenario: To query an existing account, wrong height
```

Comment:
Scenario titles could be simplified to drive more accuracy and readability. Strive to be consistent with what you have in accounts.feature. It reads more naturally since you have a colon in between the Scenario keyword and the title. For example:

Instead of "Scenario to query an existing account", we could use "Scenario: Query an existing account with wrong height".

Reply:
Titles lifted in the style of the [func tests](https://github.com/pokt-network/pocket-core-func-tests/blob/65fc1cc7a9093b0fcbaab0f52ed3f345827c4de8/tests/cli-commands/pocket-query-commands.feature#L51); will take a look at all scenario titles, thanks for pointing it out.

Reply:
As mentioned elsewhere -- the style was lifted from the original [functional test suite](https://github.com/pokt-network/pocket-core-func-tests/blob/65fc1cc7a9093b0fcbaab0f52ed3f345827c4de8/tests/cli-commands/pocket-query-commands.feature#L51). Will look at all scenario titles.